acestep/inference.py

```
@@ -331,6 +332,42 @@ def _update_metadata_from_lm(
+        logger.warning("llm_handler.unload() failed: {}", exc)
+
+    gc.collect()
+
+    try:
+        if torch.cuda.is_available():
+            torch.cuda.synchronize()
+            torch.cuda.empty_cache()
+            try:
+                torch.cuda.ipc_collect()
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)